source-mongodb/backfill.go

```diff
@@ -141,7 +141,7 @@ func (c *capture) doBackfill(
 		"collection":         binding.resource.Collection,
 		"estimatedTotalDocs": estimatedTotalDocs,
 	})
-	logEntry.Info("starting backfill for collection")
+	logEntry.Debug("starting backfill for collection")
```

Comment:
Is this only logged once per task startup? If yes I'd keep it at INFO

Reply:
These get logged every 5 minute cycle so it's pretty spammy. We get the info log about having progressed the backfill for the collection at the end of every cycle, and an info log when a collection backfill completes, so it was a lot of noise.